test/deploy.sh

```diff
@@ -131,8 +135,23 @@ $DRYRUN $ROOT/bin/beacon -numShards $SHARDS > $log_folder/beacon.log 2>&1 | tee
 sleep 1 #waiting for beaconchain
 MA=$(grep "Beacon Chain Started" $log_folder/beacon.log | awk -F: ' { print $2 } ')
 
+echo "launching boot node ..."
+$DRYRUN $ROOT/bin/bootnode > $log_folder/bootnode.log 2>&1 | tee -a $LOG_FILE &
+sleep 1
+BN_MA=$(grep "BN_MA" $log_folder/bootnode.log | awk -F\= ' { print $2 } ')
+
+HMY_OPT=
+HMY_OPT2=
```

Comment:
Maybe use `HMY_OPT+=...` and do away with this extra variable?

Reply:
HMY_OPT2 is not applicable to txgen, that's why I have to use two variables here.